temporian/core/operators/propagate.py

```diff
@@ -132,12 +116,19 @@ def propagate(
     Args:
         input: Node to propagate.
         sampling: Index to propagate over.
+        resample: If true, apply a tp.resample operator. In this case, the
+          output of `propagate` has the same sampling as `sampling`.

```

Comment:
That is a good point, but my interpretation of this argument would be that it takes a list of feature names instead of an even set (like add_index). This alternative propagate might actually be a good idea (possibly a replacement of this one). I'll make some more tests, and make a proposal in a following meeting.

For the time being, I am just adding a todo.